### PR TITLE
ACL Implementation 

### PR DESCRIPTION
**Pull request by andresgnlez**

### CODE REVIEW

**Comment by tiagojsag** [comment]
@andresgnlez need your help to style something, let's talk tomorrow

File: app/src/actions/user.js [hunk 1]
```diff
@@ -10,6 +12,14 @@ export function setToken(token) {
   };
 }
 
+function getUserData(data) {
+  return data;
```
Comment (nerik) [inline app/src/actions/user.js]: It's a no-op, what is it for?

File: app/src/containers/Map.js [hunk 2]
```diff
@@ -28,7 +28,8 @@ const mapStateToProps = (state) => ({
   activeBasemap: state.map.activeBasemap,
   layerModal: state.map.layerModal,
   supportModal: state.map.supportModal,
-  layerLibraryModal: state.map.layerLibraryModal.open
+  layerLibraryModal: state.map.layerLibraryModal.open,
+  userPermissions: state.user.acl
```
Comment (nerik) [inline app/src/containers/Map.js]: Why calling this `userPermissions` in containers/components, and `acl` in the app state?

**Comment by nerik** [comment]
@tiagojsag just two comments, otherwise lgtm 👍

**Pull request merged**